x/incentive/keeper/msg_server_earn_test.go

```diff
@@ -214,10 +214,24 @@ func (suite *HandlerTestSuite) TestEarnLiquidClaim() {
 		AmountOf("ukava").
 		Mul(sdk.NewDec(99)).
 		Quo(sdk.NewDec(100)).
-		TruncateInt()
-	suite.BalanceEquals(userAddr2, preClaimBal2.Add(sdk.NewCoin("ukava", stakingRewards2)))
+		RoundInt()
+
+	suite.BalanceInEpsilon(
+		userAddr2,
+		preClaimBal2.Add(sdk.NewCoin("ukava", stakingRewards2)),
+		// Highest precision to allow 1ukava margin of error
+		// 820778117815 vs 820778117814
+		1e-11,
```

Comment:
Do you know where this change in balance came from?

Reply:
Haven't pinned it 100% down to where exactly, but the total ukava supply changed because of the additional initial validator, and there is bankers rounding when claiming which could cause changes of 1 - along with the `stakingRewards2` in the test being calculated by 99% of the total directly instead of through the incentive shares